src/EnergyPlus/DataGlobalConstants.hh

```diff
@@ -151,27 +151,7 @@ namespace Constant {
         Num
     };
 
```

Comment:
In the Pollution module, the Pollutant list also includes Water for some reason. Whatever that reason is I needed a second enumeration that had Water as one of the Pollutants. After creating that one, I realized that this one is not used for anything so I deleted it.

Reply:
Nice